src/lxcfs.c

```diff
@@ -770,27 +786,33 @@ static int lxcfs_access(const char *path, int mode)
 static int lxcfs_releasedir(const char *path, struct fuse_file_info *fi)
 {
 	int ret;
+	enum lxcfs_virt_t type;
 
-	if (strcmp(path, "/") == 0)
-		return 0;
+	type = file_info_type(fi);
 
-	if (strncmp(path, "/cgroup", 7) == 0) {
+	if (LXCFS_TYPE_CGROUP(type)) {
 		up_users();
 		ret = do_cg_releasedir(path, fi);
 		down_users();
 		return ret;
 	}
 
-	if (strcmp(path, "/proc") == 0)
-		return 0;
-
-	if (strncmp(path, "/sys", 4) == 0) {
+	if (LXCFS_TYPE_SYS(type)) {
 		up_users();
 		ret = do_sys_releasedir(path, fi);
 		down_users();
 		return ret;
 	}
 
+	if (LXCFS_TYPE_PROC(type)) 
+		return 0;
+
+	if (path && strcmp(path, "/") == 0) 
+		return 0;
+
+	lxcfs_error("lxcfs_releasedir() unknown file type: path=%s, type=%d\n",
```

Comment:
you don't need extra `\n` as `lxcfs_error` prints a new line automatically

Reply:
Fixed in latest commit.